dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/io/FileSystemByteSink.java

```diff
@@ -30,14 +32,18 @@
 public class FileSystemByteSink extends ByteSink {
 
   private final Path path;
+  private final boolean append;
 
-  public FileSystemByteSink(@Nonnull Path path) {
+  public FileSystemByteSink(@Nonnull Path path, boolean append) {
```

Comment:
I will prefere to use `enum` instead of `boolean` with explanation comment during call.